tests/test_diaForcedSource.py

```diff
@@ -0,0 +1,207 @@
+import lsst.utils.tests
+
+
+def create_test_dia_objects(n_points, wcs, startPos=100):
+    """Create dummy DIASources or DIAObjects for use in our tests.
+    Parameters
+    ----------
+    n_points : `int`
+        Number of DiaObject test points to create.
+    wcs : `lsst.afw.geom.SkyWcs`
+        Wcs to convert RA/Dec to x/y if provided.
+    startPos : `int`
+        Start position to iterate from when creating test DiaObjects
+
+    Returns
+    -------
+    test_points : `lsst.afw.table.SourceCatalog`
+        Catalog of points to test.
+    """
+    objects = afwTable.SourceCatalog(make_dia_object_schema())
+
+    for src_idx in range(n_points):
+        src = objects.addNew()
+        src['id'] = src_idx
+        src.setCoord(wcs.pixelToSky(startPos + src_idx,
+                                    startPos + src_idx))
+    return objects
+
+
+class TestDiaForcedSource(unittest.TestCase):
+
+    def setUp(self):
+        # CFHT Filters from the camera mapper.
+        self.filter_names = ["u", "g", "r", "i", "z"]
+        afwImageUtils.resetFilters()
+        afwImageUtils.defineFilter('u', lambdaEff=374, alias="u.MP9301")
+        afwImageUtils.defineFilter('g', lambdaEff=487, alias="g.MP9401")
+        afwImageUtils.defineFilter('r', lambdaEff=628, alias="r.MP9601")
+        afwImageUtils.defineFilter('i', lambdaEff=778, alias="i.MP9701")
+        afwImageUtils.defineFilter('z', lambdaEff=1170, alias="z.MP9801")
+
+        # metadata taken from CFHT data
+        # v695856-e0/v695856-e0-c000-a00.sci_img.fits
+        self.metadata = dafBase.PropertySet()
+
+        self.metadata.set("SIMPLE", "T")
+        self.metadata.set("BITPIX", -32)
+        self.metadata.set("NAXIS", 2)
+        self.metadata.set("NAXIS1", 1024)
+        self.metadata.set("NAXIS2", 1153)
+        self.metadata.set("RADECSYS", 'FK5')
+        self.metadata.set("EQUINOX", 2000.)
+
+        self.metadata.setDouble("CRVAL1", 215.604025685476)
+        self.metadata.setDouble("CRVAL2", 53.1595451514076)
+        self.metadata.setDouble("CRPIX1", 1109.99981456774)
+        self.metadata.setDouble("CRPIX2", 560.018167811613)
+        self.metadata.set("CTYPE1", 'RA---SIN')
+        self.metadata.set("CTYPE2", 'DEC--SIN')
+
+        self.metadata.setDouble("CD1_1", 5.10808596133527E-05)
+        self.metadata.setDouble("CD1_2", 1.85579539217196E-07)
+        self.metadata.setDouble("CD2_2", -5.10281493481982E-05)
+        self.metadata.setDouble("CD2_1", -8.27440751733828E-07)
+
+        self.wcs = afwGeom.makeSkyWcs(self.metadata)
+
+        self.flux0 = 10000
+        self.flux0_err = 100
+        self.exposureId = 1234
+        self.exposureTime = 200.
+        self.imageSize = [1024, 1153]
+        self.dateTime = "2014-05-13T17:00:00.000000000"
+
+        # Make images  with one source in them and distinct values and
+        # variance for each image.
+        # Direct Image
+        ms = afwImage.MaskedImageF(lsst.geom.ExtentI(self.imageSize[0] + 1,
```

Comment:
"ms" is not a very descriptive variable name. Is the "m" for masked image and the "s" for source? I'd prefer to see a slightly longer name, but I guess this is ok for a test.

Reply:
sorry, these were copy-pasted from a different unittest. I'll give them slightly more descriptive names.